docs/binary-packages.md

```diff
@@ -5,28 +5,9 @@ layout: documentation
 
 # Binary Packages
 
-## Downloading the Mesos RPM
+## Downloading the Mesos binaries
 
-Download and install the latest stable RPM binary from the [Bintray Repository](https://bintray.com/apache/mesos/):
-
-    $ cat > /tmp/bintray-mesos-el.repo <<EOF
-    #bintray-mesos-el - packages by mesos from Bintray
-    [bintray-mesos-el]
-    name=bintray-mesos-el
-    baseurl=https://dl.bintray.com/apache/mesos/el7/x86_64
-    gpgcheck=0
-    repo_gpgcheck=0
-    enabled=1
-    EOF
-
-    $ sudo mv /tmp/bintray-mesos-el.repo /etc/yum.repos.d/bintray-mesos-el.repo
-
-    $ sudo yum update
-
-    $ sudo yum install mesos
-
-The above instructions show how to install the latest version of Mesos for RHEL 7.
-Substitute `baseurl` the with the appropriate URL for your operating system.
+Download and install the latest stable binary: [Downloads](https://mesos.apache.org/downloads/)
 
 ## Start Mesos Master and Agent.
```

Comment:
Not yours, but I think we do no need the trailing period.

Reply:
Sorry, I don't understand. :sweat_smile:  Do you mean we don't need "Start Mesos Master...."?

Reply:
I mean it should be `## Start Mesos Master and Agent` rather than `## Start Mesos Master and Agent.`.

Reply:
Ah ok I got it. :-) Thanks.